source/geometries/GenericWLSFiber.cc

```diff
@@ -0,0 +1,298 @@
+// -----------------------------------------------------------------------------
+//  nexus | GenericWLSFiber.cc
+//
+//  Geometry of a configurable wavelength shifting optical fiber.
+//
+//  The NEXT Collaboration
+// -----------------------------------------------------------------------------
+
+#include "GenericWLSFiber.h"
+
+#include "MaterialsList.h"
+#include "OpticalMaterialProperties.h"
+#include "Visibilities.h"
+
+#include <G4Tubs.hh>
+#include <G4Box.hh>
+#include <G4Colour.hh>
+#include <G4LogicalVolume.hh>
+#include <G4PVPlacement.hh>
+#include <G4Material.hh>
+#include <G4GenericMessenger.hh>
+#include <G4OpticalSurface.hh>
+#include <G4LogicalSkinSurface.hh>
+
+
+using namespace nexus;
+
+
+GenericWLSFiber::GenericWLSFiber(G4String    name,
+                                 G4bool      isround,
+                                 G4double    thickness,
+                                 G4double    length,
+                                 G4bool      doubleclad,
+                                 G4bool      with_coating,
+                                 G4Material* coating_mat,
+                                 G4Material* core_mat,
+                                 G4bool      visibility):
+  GeometryBase    (),
+  name_           (name),
+  isround_        (isround),         // "round"  or "square"
```

Comment:
comment is misleading here, I would say `true` if round `false` if square

Reply:
Fixed in the latest commit!